tox.ini

```diff
@@ -28,7 +28,8 @@ commands =
 
 deps =
     mypy-lang
-    typed_ast
+    typed-ast < 1.0.0
+    # git+git://github.com/python/mypy.git@3cc52013c35dadda57964c7d4ae6c664bfc1329c
```

Comment:
There's two possible strategies here:

1. Constrain `typed-ast` to a version that doesn't break the current release of MyPy; or
1. Pin MyPy to a not-yet-released commit that accommodates (and constrains) `typed-ast`.

This uses the first approach. Either way, there's work to undo the pin with the next release of MyPy.